src/datetime/serde.rs

```diff
@@ -65,13 +64,9 @@ impl<'de> de::Visitor<'de> for DateTimeVisitor {
     }
 }
 
-/// Deserialize a value that optionally includes a timezone offset in its
-/// string representation
+/// Deserialize an RFC 3339 formatted string into a `DateTime<FixedOffset>`.
```

Comment:
FWIW, it's somewhat on purpose that the first line of docstrings doesn't have a `.`. These are sentence fragments, not full sentences.

Reply:
Really? Okay.